### PR TITLE
Update day10.md

### PR DESCRIPTION
**Pull request by EugeneFlesselle**

### CODE REVIEW

**Review by bishabosha — APPROVED** [review]
looks great, I just added a few formatting changes

File: docs/2023/puzzles/day10.md [hunk 1]
```diff
@@ -132,8 +138,12 @@ def findLoop(grid: Seq[String]): Seq[(Int, Int)] =
     val startI = grid.indexWhere(_.contains('S'))
     (startI, grid(startI).indexOf('S'))
 
-  /** List of connected points starting from 'S' (p0, p1) :: (p1, p2) :: (p2, p3) :: ... */
-  val loop = LazyList.iterate((start, connected(grid)(start).head)): (prev, curr) =>
+  val initial = (start, connected(grid)(start).head)
+
+  /** List of connected points starting from 'S'
+   * e.g. `(y0, x0) :: (y1, x1) :: (y2, x2) :: ...`
```
Comment (EugeneFlesselle) [inline docs/2023/puzzles/day10.md:144]: If we want to show both coordinates then it should be something like this:
`((y0, x0), (y1, x1)) :: ((y1, x1), (y2, x2)) :: ((y2, x2), (y3, x3)) :: ...`

Comment (EugeneFlesselle) [reply docs/2023/puzzles/day10.md:144]: Not supper important, but we could probably still fix it
@bishabosha

Comment (bishabosha) [reply docs/2023/puzzles/day10.md:144]: Oh right it's connected points oops

**Pull request merged**